test/functionalTest/cases/1585_log_summary/log_summary.test

```diff
@@ -201,11 +190,11 @@ Date: REGEX(.*)
  Transactions: 4 (new: 4)
  DB status: ok (created: 0, removed: 0)
  Notification failure active alarms: 0 (created: 0, removed: 0)
- Bad input active alarms: 1 (created: 6, removed: 0)
+ Bad input active alarms: 1 (created: 1, removed: 0)
  Transactions: 5 (new: 1)
  DB status: ok (created: 0, removed: 0)
  Notification failure active alarms: 0 (created: 0, removed: 0)
- Bad input active alarms: 1 (created: 7, removed: 0)
+ Bad input active alarms: 1 (created: 1, removed: 0)
```

Comment:
Step 7 is a wrong operation, but associated also to client 0.0.0.0 (i.e. an alarm that already exist, as it has not be released). Thus, this should be:

"Bad input active alarms: 1 (created: 0, removed: 0)"